features/functests/sctp/sctp.go

```diff
@@ -1,92 +1,57 @@
-package test_test
+package sctp
```

Comment:
does this file need to be ```sctp_test.go```? any thoughts on whether it belongs in the features/functests dir rather than a subdirectory?

Reply:
> does this file need to be `sctp_test.go`? any thoughts on whether it belongs in the features/functests dir rather than a subdirectory?

We can't make it a `_test.go` file since no go tests are in there, and the compiler would complain.

As per the folder question, it's not about folders but it's about packages. 
I think that having different feature tests to belong to different packages will provide a more organic structure. The tests will be isolated from each other and will need to import only what they need, as opposed to having everything in the same package sharing all the functions and all the variables.